js/deribit.js

```diff
@@ -0,0 +1,303 @@
+        return {
+            'currency': 'BTC',
+            'address': account['depositAddress'],
+            'status': 'ok',
+            'info': account,
+        };
+    }
+
+    parseTrade (trade, market) {
+        return {
+            'info': trade,
+            'timestamp': trade['timeStamp'],
+            'datetime': this.iso8601 (trade['timeStamp']),
+            'symbol': market['symbol'],
+            'id': trade['tradeId'].toString (),
+            'order': undefined,
+            'type': undefined,
+            'side': trade['direction'],
+            'price': this.safeFloat (trade, 'price'),
+            'amount': this.safeFloat (trade, 'quantity'),
+        };
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'instrument': market['id'],
+        };
+        if (typeof limit !== 'undefined') {
+            request['limit'] = limit;
+        } else {
+            request['limit'] = 10000;
+        }
+        let response = await this.publicGetGetlasttrades (this.extend (request, params));
+        return this.parseTrades (response['result'], market, since, limit);
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let response = await this.publicGetGetorderbook ({ 'instrument': market['id'] });
+        let timestamp = response['usOut'] / 1000000;
+        let orderbook = this.parseOrderBook (response['result'], timestamp, 'bids', 'asks', 'price', 'quantity');
+        return this.extend (orderbook, {
+            'nonce': this.safeInteger (response, 'tstamp'),
+        });
+    }
+
+    parseOrder (order) {
+        return {
+            'info': order,
+            'id': order['orderId'].toString (),
+            'timestamp': order['created'],
+            'datetime': this.iso8601 (order['created']),
+            'lastTradeTimestamp': undefined,
+            'symbol': order['instrument'],
+            'type': undefined,
+            'side': order['direction'].toLowerCase (),
+            'price': order['price'],
+            'amount': order['quantity'],
+            'cost': order['commission'],
+            'filled': order['filledQuantity'],
+            'remaining': order['quantity'] - order['filledQuantity'],
+            'status': order['state'],
+            'fee': undefined,
+        };
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        let response = await this.privateGetOrderstate ({ 'orderId': id });
+        return this.parseOrder (response['result']);
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let response = await this.privateGetTradeHistory ({ 'instrument': symbol });
+        return this.parseOrders (response['result']);
```

Comment:
This line confuses trades with orders... Should be `parseTrades` here...

https://github.com/ccxt/ccxt/wiki/Manual#how-orders-are-related-to-trades

Reply:
Can you, plz, edit this + `parseTrade` to support for fetchMyTrades? ) Thx!